source/features/quick-repo-deletion.tsx

```diff
@@ -83,12 +83,12 @@ async function start(buttonContainer: HTMLDetailsElement): Promise<void> {
 	select('.btn', buttonContainer)!.textContent = 'Deleting repo…';
 
 	try {
-		const {nameWithOwner} = getRepo()!;
+		const {nameWithOwner, owner} = getRepo()!;
 		await api.v3('/repos/' + nameWithOwner, {
 			method: 'DELETE',
 			json: false
 		});
-		addNotice(`Repository ${nameWithOwner} deleted`, {action: false});
+		addNotice(<span>Repository {nameWithOwner} deleted. You can <a href="https://docs.github.com/en/free-pro-team@latest/github/administering-a-repository/restoring-a-deleted-repository">restore it</a> or see <a href={`/${owner}?tab=repositories&q=&type=fork`}>your other forks</a></span>, {action: false});
```

Comment:
Does `owner` also work for organizations? Also maybe we can provide a direct link for recovery if the link is the same for both users and orgs (via `owner` again)

Reply:
I checked now that I'm on the computer. Both URLs vary depending on user vs org:

Deleted repos:

```
https://github.com/settings/deleted_repositories
https://github.com/organizations/GhostText/settings/deleted_repositories
```

Fork list:

```
https://github.com/fregante?tab=repositories&type=fork
https://github.com/GhostText?type=fork
```

The issue is that as far as I know it's not possibly to detect whether a repo belongs to an organization, at least this was the case the last time we needed this piece of information.

For the forks list however, the first actually works for both anyway, `tab` is discarded.

So I think this is ok as is, unless you figure out how to detect an org repo without HTTP calls.

Reply:
$('[data-owner-scoped-search-url]')

Reply:
Great! It's worth adding a `isOrganizationRepo` to detection module. The fork list link can still stay as is though.

Reply:
I'm vacationing over the weekend, so either I will do it tonight or next week.